shukla/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu
+RUN apt-get update && \
+	DEBIAN_FRONTEND=non-interactive \
+	apt-get -y install default-jre-headless && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install make
+CMD ["make", "run"]
+CMD ["make", "clean"]
```

Comment:
There can be only one `CMD` per Dockerfile, so the last one effectively clobbers the first. So at runtime only 

```
make clean
```

will get run.